### PR TITLE
[confluent-local] be more explicit about confluent local in `confluent local start/stop`

### PR DESCRIPTION
**Pull request by MuweiHe**
Release Notes
-------------

Breaking Changes
- PLACEHOLDER

New Features
- PLACEHOLDER

Bug Fixes
- PLACEHOLDER

Checklist
---------
1. [CRUCIAL] Is the change for features that are already live in prod?
   * yes: ok

What
----

When user already has one container running, prompt them if they want a new session. if yes, clean up the current one and create a new one.

Improved wording to be more instructive. 

References
----------

Test & Review
-------------

Open Questions / Follow-ups
---------------------------

### CODE REVIEW

File: internal/pkg/errors/error_message.go [hunk 1]
```diff
@@ -218,9 +218,9 @@ const (
 	MacVersionErrorMsg               = "macOS version >= %s is required (detected: %s)"
 	JavaExecNotFondErrorMsg          = "could not find java executable, please install java or set JAVA_HOME"
 	NothingToDestroyErrorMsg         = "nothing to destroy"
-	ConfluentLocalStartedErrorMsg    = "Confluent Local container has already been created"
+	ConfluentLocalStartedErrorMsg    = "Confluent Local is already running"
```
Comment (brianstrauch) [inline internal/pkg/errors/error_message.go:221]:
```suggestion
	ConfluentLocalIsAlreadyRunningErrorMsg    = "Confluent Local is already running"
```

File: internal/pkg/errors/error_message.go [hunk 2]
```diff
@@ -218,9 +218,9 @@ const (
...
 	ConfluentLocalStartedSuggestions = "Continue your experience with Confluent Local running `confluent local kafka produce` and `confluent local kafka consume`.\n" +
-		"To stop Confluent Local container, run `confluent local kafka stop` or `docker container rm confluent-local -f`."
+		"To start a fresh Confluent Local session, run `confluent local kafka stop`, then run `confluent local kafka start`."
```
Comment (brianstrauch) [inline internal/pkg/errors/error_message.go:223]:
```suggestion
		"To start a new Confluent Local session, run `confluent local kafka stop`, then run `confluent local kafka start`."
```

File: internal/cmd/local/command_kafka_stop.go [hunk 3]
```diff
@@ -51,7 +51,7 @@ func (c *Command) kafkaStop(cmd *cobra.Command, args []string) error {
 			}
 			log.CliLogger.Tracef("Confluent Local container removed")
 
-			output.Printf("Stopped Confluent Local container %v.\n", getShortenedContainerId(container.ID))
+			output.Printf("Confluent Local has been stopped: removed container %v.\n", getShortenedContainerId(container.ID))
```
Comment (brianstrauch) [inline internal/cmd/local/command_kafka_stop.go:54]: I assume this is a string?
```suggestion
			output.Printf("Confluent Local has been stopped: removed container \"%s\".\n", getShortenedContainerId(container.ID))
```

File: internal/pkg/errors/error_message.go [hunk 4]
```diff
@@ -214,13 +214,13 @@ const (
...
-	JavaExecNotFondErrorMsg          = "could not find java executable, please install java or set JAVA_HOME"
-	NothingToDestroyErrorMsg         = "nothing to destroy"
-	ConfluentLocalStartedErrorMsg    = "Confluent Local is already running"
-	ConfluentLocalStartedSuggestions = "Continue your experience with Confluent Local running `confluent local kafka produce` and `confluent local kafka consume`.\n" +
-		"To start a fresh Confluent Local session, run `confluent local kafka stop`, then run `confluent local kafka start`."
+	NoLogFoundErrorMsg                     = "no log found: to run %s, use `confluent local services %s start`"
+	MacVersionErrorMsg                     = "macOS version >= %s is required (detected: %s)"
+	JavaExecNotFondErrorMsg                = "could not find java executable, please install java or set JAVA_HOME"
+	NothingToDestroyErrorMsg               = "nothing to destroy"
+	ConfluentLocalIsAlreadyRunningErrorMsg = "Confluent Local is already running"
```
Comment (brianstrauch) [inline internal/pkg/errors/error_message.go:221]: If this constant is only being used in one place, it doesn't need to be public/global.

**Comment by MuweiHe** [comment]
@brianstrauch i think this pr isn't to be more explicit about docker. we still keep the container id etc but mention more about kafka. wdyt?

**Comment by brianstrauch** [comment]
> wdyt?

It seemed to me like the wording was more explicit here: https://github.com/confluentinc/cli/pull/2090/files#diff-265b5577d5f85e9146139535dc982d25fc3d6d41b01c374cf9907a014729df89R54

Feel free to change it back if you disagree, just changed it because I thought it was a typo!

File: internal/pkg/errors/catcher.go [hunk 5]
```diff
@@ -385,7 +385,8 @@ func CatchContainerNameInUseError(err error) error {
 	}
 
 	if strings.Contains(err.Error(), "The container name \"/confluent-local\" is already in use") {
-		return NewErrorWithSuggestions(ConfluentLocalIsAlreadyRunningErrorMsg, ConfluentLocalStartedSuggestions)
+		return NewErrorWithSuggestions("Confluent Local is already running", "Continue your experience with Confluent Local running `confluent local kafka produce` and `confluent local kafka consume`.\n"+
```
Comment (brianstrauch) [inline internal/pkg/errors/catcher.go:400]:
```suggestion
		return NewErrorWithSuggestions("Confluent Local is already running", "Continue your experience with Confluent Local by running `confluent local kafka produce` and `confluent local kafka consume`.\n"+
```

File: test/local_test.go [hunk 6]
```diff
@@ -3,16 +3,15 @@ package test
...
 	"github.com/stretchr/testify/require"
 )
 
 func (s *CLITestSuite) TestLocalKafka() {
-	if runtime.GOOS == "darwin" {
-		s.T().Skip()
-	}
+	// if runtime.GOOS == "darwin" {
+	// 	s.T().Skip()
+	// }
```
Comment (brianstrauch) [inline test/local_test.go]:
```suggestion
	if runtime.GOOS == "darwin" {
		s.T().Skip()
	}
```

File: internal/cmd/local/command_kafka_start.go [hunk 7]
```diff
@@ -57,6 +58,33 @@ func (c *Command) kafkaStart(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	containers, err := dockerClient.ContainerList(context.Background(), types.ContainerListOptions{All: true})
+	if err != nil {
+		return err
+	}
+
+	for _, container := range containers {
+		if container.Image == dockerImageName {
```
[the remark this answers is not in the document]
Comment (MuweiHe) [reply internal/cmd/local/command_kafka_start.go:67]: sadly no..

File: internal/cmd/local/command_kafka_start.go [hunk 8]
```diff
@@ -57,6 +58,33 @@ func (c *Command) kafkaStart(cmd *cobra.Command, args []string) error {
...
+			output.Printf("Confluent Local is already running.\n")
```
Comment (sgagniere) [inline internal/cmd/local/command_kafka_start.go:68]:
```suggestion
			output.Println("Confluent Local is already running.")
```
Small nitpick.

**Review by sgagniere — APPROVED** [review]

**Review by DABH — APPROVED** [review]

**Pull request merged**